### ISSUE TITLE
`du -sh` hangs the shell

### ISSUE THREAD

**Issue by jaclu** [blinksh/blink#1914]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

There was another bug report #1422 that did mention the du -sh issue, but in that case, just as I did in my (closed) #1342 this specific problem was just mentioned in passing. When I asked the tech that closed #1342 why it was closed without addressing this issue, they just replied I didn't notice that part.
Not intended to complain about that closing, it made sense. But since the only other mention of du -sh was similarly hidden away in a long-forgotten bug report. It seems reasonable to open a new one dedicated to this :)

### Configuration

The current AppStore version as of the date of this Bug report

### Describe the bug

TLDR: typing `du -sh` in any form that I have tried hangs that shell forever. The App remains usable and other tabs are not impacted.

I tried a few variations to see if it was related to pwd or if a folder param was given

## $HOME - ok not entirely but you get the point...
``` shell
cd
du -sh
#  shell hangs forever
```

## cd to folder
``` shell
cd
cd .ssh
du -sh
#  shell hangs forever
```

## give folder as a parameter
``` shell
cd
du -sh .ssh
#  shell hangs forever
```

A normal 'du -sh' response

**Comment by carloscabanero** [comment]
Done 17.3.0
